tracer/test/Datadog.Trace.Debugger.IntegrationTests/ProbesTests.cs

```diff
@@ -311,15 +311,28 @@ async Task RunPhase(ProbeDefinition[] snapshotProbes, ProbeAttributeBase[] probe
 
                 if (metricProbes.Any())
                 {
-                    var requests = await agent.WaitForStatsdRequests(metricProbes.Length);
-                    requests.Should().OnlyContain(s => s.Contains($"service:{EnvironmentHelper.SampleName}"));
+                    var expectedNumberOfSnapshots = DebuggerTestHelper.CalculateExpectedNumberOfSnapshots(metricProbes);
 
-                    foreach (var probeAttributeBase in metricProbes)
+                    if (expectedNumberOfSnapshots > 0)
                     {
-                        var metricName = (probeAttributeBase as MetricMethodProbeTestDataAttribute)?.MetricName ?? (probeAttributeBase as MetricLineProbeTestDataAttribute)?.MetricName;
-                        var req = requests.SingleOrDefault(r => r.Contains(metricName));
-                        Assert.NotNull(req);
-                        req.Should().Contain($"service:{EnvironmentHelper.SampleName}");
+                        // meaning there is an error so we don't receive metrics but evaluation error
+                        var snapshots = await agent.WaitForSnapshots(expectedNumberOfSnapshots);
+                        Assert.Equal(expectedNumberOfSnapshots, snapshots?.Length);
+                        await ApproveSnapshots(snapshots, testDescription, isMultiPhase, phaseNumber);
```

Comment:
Where is the snapshot for the error that's supposedly happening in `MetricCountNonNumeric`? I can't seem to find it, did you perhaps forget to push it?

Reply:
Found it - we scrub the error message in the verified files, which made it difficult to notice that it's there.